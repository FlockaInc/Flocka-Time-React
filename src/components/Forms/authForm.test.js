import React from 'react';
import { render, cleanup } from '@testing-library/react';
import AuthForm from './authForm';

afterAll(cleanup);

it('renders with sign in button', () => {
  const { getByTestId } = render(<AuthForm authType='signin' />);

  expect(getByTestId('submitBtn')).toHaveTextContent('Sign In');
});

it('renders with sign up button', () => {
  const { getByTestId } = render(<AuthForm authType='signup' />);

  expect(getByTestId('submitBtn').toHaveTextContent('Sign Up'));
});