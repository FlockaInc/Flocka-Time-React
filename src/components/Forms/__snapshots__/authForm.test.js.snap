// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <form>
    <div
      class="form-group"
    >
      <label
        for="email"
      >
        Email
      </label>
      <input
        class="form-control"
        id="email"
        placeholder="email@example.com"
        type="email"
        value=""
      />
    </div>
    <div
      class="form-group"
    >
      <label
        for="password"
      >
        Password
      </label>
      <input
        class="form-control"
        id="password"
        placeholder="password"
        type="password"
        value=""
      />
    </div>
    <button
      class="btn btn-primary"
      type="submit"
    >
      Sign In
    </button>
    <button
      class="btn fb-btn"
      type="submit"
    >
      Facebook
    </button>
  </form>
</DocumentFragment>
`;
